mlflow/metrics/metric_definitions.py

```diff
@@ -114,7 +114,7 @@ def _cached_evaluate_load(path: str, module_type: str = "metric"):
 
     try:
         return evaluate.load(path, module_type=module_type)
-    except FileNotFoundError:
+    except (FileNotFoundError, OSError):
```

Comment:
This is a fix for https://github.com/mlflow/mlflow/actions/runs/14466032076/job/40568503645:

```
2025/04/15 09:37:17 WARNING mlflow.metrics.metric_definitions: Failed to load 'toxicity' metric (error: OSError("We couldn't connect to 'https://huggingface.co/' to load the files, and couldn't find them in the cached files.\nCheckout your internet connection or see how to run the library in offline mode at 'https://huggingface.co/docs/transformers/installation#offline-mode'.")), skipping metric logging.
```